.gitpod.yml

```
@@ -12,6 +12,7 @@ tasks:
       cd python/
       python3.9 -m venv venv --system-site-packages
       source venv/bin/activate
+      python -m pip install --upgrade pip
       pip install -e .[all]
     command: cd /workspace/AaC/python/; source venv/bin/activate # Always launch a console with our virtual env
```

Looks like this path got missed in being updated. 

```suggestion
    command: cd /workspace/aac-spec/python/; source venv/bin/activate # Always launch a console with our virtual env
```